strings.json

```
@@ -33,66 +36,77 @@
     "zh-cn": "filter=\"id:-{{id}}+tags:hash-zh-cn\"",
     "ko": "filter=\"id:-{{id}}+tags:hash-ko\"",
     "es": "filter=\"id:-{{id}}+tags:hash-es\"",
-    "tr": "filter=\"id:-{{id}}+tags:hash-tr\""
+    "tr": "filter=\"id:-{{id}}+tags:hash-tr\"",
+    "pt-br": "filter=\"id:-{{id}}+tags:hash-pt-br\""
   },
   "global.footer.from_the_blog": {
-    "en": "From the blog"
+    "en": "From the blog",
+    "pt-br": "Do Blog"
   },
   "global.footer.get_involved": {
     "en": "Want to get involved?",
     "ru": "Заинтересован?",
     "ko": "폴카닷과 함께 하고 싶으신가요?",
     "es": "¿Quiere colaborar?",
-    "tr": "Katılmak ister misin?"
+    "tr": "Katılmak ister misin?",
+    "pt-br": "Quer colaborar?"
   },
   "global.footer.join_community": {
     "en": "Join the Community",
     "ru": "Присоединяйтесь к сообществу",
     "ko": "폴카닷 커뮤니티에 합류하시고,",
     "es": "Únase a la comunidad",
-    "tr": "Topluluğa Katıl"
+    "tr": "Topluluğa Katıl",
+    "pt-br": "Participe da Comunidade"
   },
   "global.footer.learn": {
     "en": "or learn about how to {{build_link}}.",
     "ru": "или узнайте, как {{build_link}}",
     "ko": "{{build_link}}",
     "es": "o aprenda sobre cómo {{build_link}}.",
-    "tr": "veya geliştirmeyi öğren {{build_link}}."
+    "tr": "veya geliştirmeyi öğren {{build_link}}.",
+    "pt-br": "ou aprenda como {{build_link}}"
   },
   "global.footer.learn.build_link": {
     "en": "build on Polkadot",
     "ru": "работать с Polkadot",
     "ko": "폴카닷을 어떻게 개발에 활용할 수 있는지 확인해보세요.",
     "es": "desarrolle en Polkadot",
-    "tr": "Polkadot üzerinde geliştir"
+    "tr": "Polkadot üzerinde geliştir",
+    "pt-br":"programe em Polkadot"
```

WDYT about this translation?

```suggestion
    "pt-br": "desenvolva no Polkadot"
```